lib/interactive.dart

```diff
@@ -194,9 +198,53 @@ class Interactive {
       peopleByLabel[key].add(people);
     });
 
+    for (var i = 0; i < xAxisCategories.length; ++i) {
+      var colSVG = svg.SvgElement.tag("g");
```

Comment:
What does this do?

Reply:
This is a group where I add all squares of each column - a group so I can do background highlights potentially later